pillar/covid19.sls

```diff
@@ -3,8 +3,56 @@ ssh:
     # Young Innovations
     # Bigyan
     - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCmcdq0cl6Gob/Z6dvto8Vev0iadUhQ/+ZkxMK4BWFz081zz5bf6LcYxdXbF90Co+wn/mIX6Dszrf5xbZ8wJYdsYhO61gsKx+jmeFiFDA3saMjXacAyFep2KropIEY3dCj+FF2wy56sV65CsZshhSY7UqQ7EnMiGU0ZoKrr4ra3hMaebZuFXRQi0bP837YGUXyVN1jm0sfKS/Tq6kmFFyk4UscgKj/G9fec10LMcfODwAz5o5+u1LL6AKQ0d92ShC6JkzTxUWcZr/o3DdhwlpKa1GdYYRbjc301IXyT1AuXd4o3Fft3C6u0O4G/NP832QvYvaNom2uToe4H/nRFZ8ZD
-    - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC+uCezwsuwpoorNNjLlfMLyii4z/gDP+ZCF6cOppyTkK4vvLU3VUr9yi2EqlL+AcOAlMtmhaTyc53XW86v8VYkyI0uzClZ4pbY4686QQoOnCFGh6nkb5RS1Y73b33ZEv9amPbJuy1VYCAIWuJ4fXXmaMxdvgCkkIuXOBCxFpL5n/cVj7uroBRCJaOSDvgWrOJZ4e/XzzStnvxaTjKv+D/if3eXhaZDbr/i7Uuv8udUQiQlnOkEmOcz5kbPE27tzVDPAVotF2YXmUycYhH0s5F2XPivhDfNdx53L8ey07NMuyD+iHYbLla3gcqwIUxqRqlEwYZ2Ki/HBNa89L912J1PWpyWExzBzOuesHVYTNYgyiLQl/ekNRlz67BhBo6puHjaH1sSCDjL/UamEG5MDwqef+Gxk233wKSNsxuRiEsE4GREwPCqaS8FPqF6wNyZDfLlr8vy3m0K8EG6ABVyBvYTvMbDOU7hDQ8kxtSKFbaZyrk204Me+NvJwpChS8tuNXc=
     # Bikram
     - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDAkUgLckYEe2PoseA+l5ekUDldHj5bLOfEMelZfi9pXitqUtekzOAJhbgcPZDh6tNw804Bs4pA44nry/T17UImoKxN5jhPGsxveiiYIyykHaX83woZm5RAunhDP2+guy9EFa6cVY+3HEd6o2oiptXk50gCzhPgOjO8G4ufaPBFAu2gfUnJB5X+mDkwH6RlK5UKdLwc7X0GO4/a3nIu3ERp3b8vC6gIvwm49x//IA3zyKpRqncuRS7B7qrQYmYbe2okNYIhMauwhsFPCzN7XQDnMhtMpt3NKHM38qNAs6hkyIofQu0kpydRz0LSigVFFcXkRgXRZqKLia7Gf5gOqYJZR4ykmmrF0ktStiG6kMWORberfKCT+R4QAUvmNCxJ9X05IQkfQGUKfoXigZ2xyHhpvQ8L3lcc010zf/HSBTfFrblnY3X0rffZfMC9g3ErODZ9F+yr3U+XtqN06snCJDyDkX6ECD5jClDhOcmWqzfbRJTliJApLcR2FO7uhKsgR4fAXCzSe7j7pKrwin1X8HjaZuO0tPR1T4v2l4Tt8eGm9PWTMhVmV/P7Wf0J7VUOCN1teaq+INNDCIXaJ/dqT9itEJbCnKxQCTY0BbaX5fNOF0jO+fi0A1fsv/GhSnZgkAno8dYLd1DcihqBvMFdnkxwveNMcVc6nDoxQ4FVjNhd7Q==
     # Kushal
     - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDn9Gw0of6Tc4e0DMRXYXZpg+scj5cefgXzCFoOxpvYtbRwomEWx4OXocHwrm5mbeZCXA6CDvSq15uH8s6vuYeyy+A+YFgBJoxVUtmfygbbHBbjQugJnY5tGu80HWmdYJJmPdFvJpf2g+DJykOtHVy4tiWR+gPhB3S/mJL4DC1FFCqJTU31QooLi/sAo6BLf5kzM49vz5e4uBumh7+SqejFTw0ONXMnDxETmHN+462QjVx2zISH6ycy7EUX8RntG3Mp7ju/i8yYE+6Lgbb+w7ozZ9fmD7GPMR+PQcX+77/I0zwaOq+YieJMgpLvApoVXqs8fUCYuDY55rXByPUir3Mz
+    # temp-salt-server
+    - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDKnNdGFL8ITQqD/BXNw9sT0SvcicV77Gr7MwiyOyhIVMbVtXXGXY7jRlly7EYc/Ejy0wGmkUtIpEOZXUbbeGyTqQOab0FU1Y1lX/ZiXMN2fYp+gMUzA7u+JMA/IpJ1CuNO7g8GpmLMs3ZqosBqkyW3mBRAQAwiWfI4z0okO0YIbzINLG9Xro+ARZ4FUhRWYe6B5Pga3rSJpQbswVghE3G96qleTqwnI++hIetiOU7w6QIbgXh4fB+EGNjh3XXV//pa69E0HvLf6O1RJXQravCDBu4rT4/KrvWcPSiGAB5dq5xsaNiVqVSqmQm1lBV84KYHzNhPREWvq/AIlKg30e1t bikramtuladhar@temp-salt-server
+apache:
+  public_access: True
+  https: True
```

Comment:
`https` is now True by default, so you only need to set it if you want it to be False. Also, it would have to be under the `apache` keys below.

Reply:
Removed https entry